### PR TITLE
Home

### PR DESCRIPTION
**Pull request by JantuDeb**
### Home Page

1.  Navbar
2. Categories
3. Banner
4. New & Trending Products

image

image

image

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+.container{
+  padding: 1.5rem 0;
+}
+.categories {
+  gap: 1rem;
+  justify-content: center;
+}
+
+.categories .card {
+  max-width: 150px;
```
Comment (t007rushi) [inline styles.css:61]: good to follow px unit for width?

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+/* Card */
+.card-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+.card-info img {
+  max-height: 220px;
```
Comment (t007rushi) [inline styles.css:94]: same as above?

File: styles.css [hunk 3]
```diff
@@ -71,32 +84,48 @@
...
+  padding: 1.5rem 0;
+  gap:1rem;
+  margin-top: 0.5rem;
+  background-color: var(--fu-red);
+  color: var(--fu-gray-100);
+}
+
+
+
 @media screen and (min-width: 830px) {
```
Comment (rkapoor10) [inline styles.css:112]: you can use ems instead of px here

**Review by rkapoor10 — APPROVED** [review]
Everything looks good. Merge it.

**Review by t007rushi — CHANGES_REQUESTED** [review]
UI looks Clean can have consistent units

**Review by psychoactiv — APPROVED** [review]

**Pull request merged**